app/sprinkles/core/assets/userfrosting/css/userfrosting.css

```diff
@@ -149,3 +161,45 @@ a .info-box {
 .btn-trash:hover {
     color: #d73925;
 }
+
+/* Custom styling for nav logo */
+.main-header .logo {
+    font-weight: 400;
+    text-align: center;
+    font-size: 18px;
+}
+
+@media(min-width: 768px) {
+    .main-header .logo {
+        text-align: left;
+    }
+}
+
+/* Padding for user menu */
+.user-menu a i {
```

Comment:
why not just put it on `.user-menu .fa` or something?

Reply:
ok